src/CBLReplicator_Internal.hh

```diff
@@ -46,7 +46,11 @@ static CBLReplicatorStatus external(const C4ReplicatorStatus &c4status) {
         (c4status.level == kC4Idle || c4status.level == kC4Stopped) && c4status.error.code == 0) {
         complete = 1.0; // When the replicator is idle or stopped, return as completed if having no changes to replicate
     } else {
-        complete = c4status.progress.unitsCompleted / std::max(float(c4status.progress.unitsTotal), 1.0f);
+        auto total = c4status.progress.unitsTotal > 0 ? c4status.progress.unitsTotal : 1;
```

Comment:
basically, you're returning std::min(completed, 1.0f)

Reply:
Yes. I was trying to avoid float comparison but probably overthinking about it.